lib/api.js

```diff
@@ -93,7 +92,7 @@ api.declare({
   route:       '/secrets/:name(*)',
   deferAuth:   true,
   name:        'remove',
-  scopes:      [['secrets:remove:<name>']],
+  scopes:      [['secrets:set:<name>']],
```

Comment:
it's also possible to do:
`[['secrets:set:<name>'], ['secrets:remove:<name>']]`

And that way have both simplicity and expressiveness... Not that I really care, we don't have use-cases for update-only or remove-only :)